docs/source/reference/structs.rst

```diff
@@ -9,6 +9,9 @@ Structs
 .. autoclass:: StudyDirection
     :members:
 
+    .. deprecated:: 1.3.0
```

Comment:
This change will first be included in the next release.
```suggestion
    .. deprecated:: 1.4.0
```

Reply:
Thank you for your comment. I will change it as you pointed.